build.boost.sh

```diff
@@ -8,8 +10,8 @@ if [[ "$OSTYPE" == "msys" ]]; then
 ./b2 --prefix=.  --layout=versioned --toolset=msvc-14.3 address-model=64 architecture=x86 variant=debug threading=multi link=static runtime-link=shared install
 elif [[ "$OSTYPE" == "darwin"* ]]; then
 
-./bootstrap.sh
-./b2 --prefix=. variant=release 
+./bootstrap.sh --prefix=`pwd`
```

Comment:
Using `pwd` works consistently. The . was acting strange...

Reply:
NOTE: AFAIK - the --prefix=`pwd` invokes the bootstrap.sh to create an include/ folder - which ./b2  --prefix=`pwd` detects consistently in the next line.